util/MsSql/entrypoint.sh

```diff
@@ -49,6 +49,12 @@ chown -R $USERNAME:$GROUPNAME /var/opt/mssql
 chown $USERNAME:$GROUPNAME /backup-db.sh
 chown $USERNAME:$GROUPNAME /backup-db.sql
 
+# Replace database name in backup-db.sql
+if [ ! -z "$DATABASE" ]
+then
+  sed -i "s/vault/$DATABASE/g" backup-db.sql
```

Comment:
Just a question, but are we sure `sed` won't change the word `vault` outside of the context of the database name? Not sure if it would exist anywhere else in the script file since I don't see any text context such as `\[vault\]` or `USE \[vault\]`, etc. which would narrow the focus down to the database name.

Reply:
I've updated the `backup-db.sql` file to use variables, and this file to use a more precise `sed` command.  Let me know what you think.